Tests/NIOHTTP1Tests/TestUtils.swift

```diff
@@ -0,0 +1 @@
+../NIOTests/TestUtils.swift
```

Comment:
Rather than create this symlink, should we just move this into a target and depend on it from the two test targets?

Reply:
@Lukasa SwiftPM doesn't unfortunately support that AFAIK. We can't make it a test target as we can't depend on that and we can't make it a normal target as it depends on `XCTest`. Therefore this is the only way I'm aware of...